x/dclauth/types/account.go

```diff
@@ -144,6 +144,17 @@ func NewPendingAccount(acc *Account, approval sdk.AccAddress, info string, time
 	return pendingAccount
 }
 
+// NewRevokedAccount creates a new RevokedAccount object.
+func NewRevokedAccount(acc *Account, approvals []*Grant, info string, time int64) *RevokedAccount {
```

Comment:
Why do we need `Information` and `time` params here? They seem to be not used...